src/nanotron/config/utils_config.py

```diff
@@ -31,6 +32,8 @@ def serialize(data) -> dict:
         value = getattr(data, field.name)
         if hasattr(value, "__dataclass_fields__"):
             result[field.name] = serialize(value)
+        elif isinstance(value, xPath):
+            result[field.name] = str(value)
```

Comment:
Why xPath ? I thought we removed it for Path

Reply:
Will still need it in s3upload to upload to s3 (with path it consider s3 path as local path i think)